api4/plugin_test.go

```diff
@@ -57,6 +57,10 @@ func TestPlugin(t *testing.T) {
 	CheckNoError(t, resp)
 	assert.Equal(t, "testplugin", manifest.Id)
 
+	// Stored in File Store: Install Plugin from URL case
+	pluginStored, _ := th.App.FileExists("./plugins/" + manifest.Id + ".tar.gz")
```

Comment:
minor nit: for completeness it might be worth asserting `err` is `nil`  here

Reply:
Fixed